07-workbase/04-schema-designer.md

```diff
@@ -0,0 +1,183 @@
+---
+sidebarTitle: Schema Designer
+pageTitle: Schema
+permalink: /docs/workbase/schema-designer
+toc: false
+---
+
+## Design a Schema
+Workbase allows the visualisation and design of a grakn schema.
+
+### Navigate to the Schema Designer
+<div class="slideshow">
+
+[slide:start]
+[body:start]![schema designer](/docs/images/workbase/1.1.1/schema_designer.png)[body:end]
+[footer:start]We can access the schema designer by clicking the top left **schema desinger** icon.[footer:end]
+[slide:end]
+
+</div>
+
+### Define New Entity Type
+<div class="slideshow">
+
+[slide:start]
+[body:start]![schema designer](/docs/images/workbase/1.1.1/define_entity_1.png)[body:end]
+[footer:start]We can use the left bar to deine new entity type[footer:end]
```

Comment:
**define**. this typo is repeated in some other footer notes.
please run grammar check on all Workbase content using Grammarly, in case we have missed other typos.

Reply:
end all footer notes with a full stop.